dmgbuild/core.py

```diff
@@ -391,14 +391,14 @@ def roundup(x, n):
                 for dirpath, dirnames, filenames in os.walk(path):
                     for f in filenames:
                         fp = os.path.join(dirpath, f)
-                        total_size += roundup(os.lstat(fp).st_size, 4096)
+                        total_size += roundup(os.lstat(fp).st_size, 4000)
             else:
-                total_size += roundup(os.lstat(path).st_size, 4096)
+                total_size += roundup(os.lstat(path).st_size, 4000)
 
         for name,target in iteritems(options['symlinks']):
-            total_size += 4096
+            total_size += 4000
 
-        total_size = str(max(total_size / 1024, 1024)) + 'K'
+        total_size = str(max(total_size / 1000, 1000)) + 'K'
```

Comment:
This, however, I can believe might be right.

Reply:
sounds good, fixed